client/src/pages/Login/Login.tsx

```diff
@@ -36,10 +37,30 @@ export default function Login(): JSX.Element {
     });
   };
 
+  const demoLogin = () => {
+    const email = 'demo@demo.com';
+    const password = 'test12345';
+    login(email, password).then((data) => {
+      if (data.error) {
+        updateSnackBarMessage(data.error.message);
+      } else if (data.success) {
+        console.log('data---', data);
```

Comment:
Could this be removed, we do not need unnecessary logs in production 😄

Reply:
removed